cms/envs/common.py

```diff
@@ -1105,6 +1105,7 @@
     # These are apps that aren't strictly needed by Studio, but are imported by
     # other apps that are.  Django 1.8 wants to have imported models supported
     # by installed apps.
+    'openedx.core.djangoapps.oauth_dispatch.apps.OAuthDispatchAppConfig',
```

Comment:
I guess this means you couldn’t mark this as lms only?

Reply:
yeah, I dug more and the only real lms only things are happening happen in the djangoapps, but nothing in the migrations that I could apply.  Makes me sad, but maybe just points towards finishing getting rid of DOP and it'll all simplify since we can pull other things out of studio and roll this back.

Reply:
👍